src/index.html

```diff
@@ -11,7 +11,156 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="index.html" method="post">
+      <fieldset class="groups">
+        <legend> Personal information </legend>
+        <label class="block" for="surname">Surname:
+          <input
+            type="text"
+            id="surname"
+            name="Surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="block" for="name">Name:
+          <input
+            type="text"
+            id="name"
+            name="Name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="block" for="age">How old are You?
+          <input
+            type="number"
+            id="age"
+            name="Age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="block" for="birth">Full date of birth:
+          <input
+            type="date"
+            id="birth"
+            name="bithday"
+            required
+          >
+        </label>
+        <label class="block" for="accept-agree">I accept  the term of the agreememt
```

Comment:
add one empty line between same level elements
```suggestion
        </label>
        
        <label class="block" for="accept-agree">I accept  the term of the agreememt
```